inflammation-analysis.py

```diff
@@ -21,7 +21,7 @@ def main(args):
     for filename in InFiles:
         inflammation_data = models.load_csv(filename)
 
-        view_data = {'average': models.daily_mean(inflammation_data), 'max': models.daily_max(inflammation_data), 'min': models.daily_min(inflammation_data)}
+        view_data = {'average': models.daily_mean(inflammation_data), 'max': models.daily_max(inflammation_data), 'min': models.daily_min(inflammation_data), **(models.s_dev(inflammation_data))}
```

Comment:
This line is too long and should be split over multiple lines for readability.
Also the **-syntax is confusing. If the changes in models.py are accepted, this would change to `'standard deviation': models.daily_stddev(inflammation_data)`

Reply:
Implemented through latest commit (bcc8a02)